src/ralph/backends/data/fs.py

```diff
@@ -224,12 +208,34 @@ def read(  # noqa: PLR0913
                 }
             )
 
+    def _iter_files_matching_query(
+        self, target: Union[str, None], query: BaseQuery
+    ) -> Iterator[Tuple[BufferedReader, Path]]:
+        """Return file/path tuples for files matching the query."""
+        if not query.query_string:
+            query.query_string = self.default_query_string
+
+        path = Path(target) if target else self.default_directory
+        if not path.is_absolute() and path != self.default_directory:
+            path = self.default_directory / path
+
+        paths = list(filter(lambda x: x.is_file(), path.glob(query.query_string)))
+        if not paths:
+            msg = "No file found for query: %s"
+            logger.info(msg, path / Path(str(query.query_string)))
```

Comment:
should this be debug ?

Reply:
Good question) As the read operation stops after this line, I think it might be useful for the caller to know why no results are yielded (without starting a debug session)